src/services/NotificationHandleService.ts

```diff
@@ -1,38 +1,42 @@
+import { showReportDialog } from '@sentry/browser'
 import { AppState } from 'react-native'
-import * as Notifications from 'expo-notifications'
-import { APP_STATE } from '../constants'
+import { APP_STATE } from './../constants'
 import ApiService from './api/ApiService'
 
 interface NotificationObject {
-  data: {
-    title: string
-    body: string
-    image: string
-  }
+  data: { [key: string]: string }
+}
+
+interface showNotificationProps {
```

Comment:
uppercased interfaces
```suggestion
interface ShowNotificationProps {
```